patches/chrome-browser-safe_browsing-download_protection-check_client_download_request.h.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/chrome/browser/safe_browsing/download_protection/check_client_download_request.h b/chrome/browser/safe_browsing/download_protection/check_client_download_request.h
+index ae219730b0e46facaeebe3192cf04ba3299aa6b6..c0524be0175aa478ac8ae28fd9b7c47f03491a5d 100644
+--- a/chrome/browser/safe_browsing/download_protection/check_client_download_request.h
++++ b/chrome/browser/safe_browsing/download_protection/check_client_download_request.h
+@@ -64,6 +64,11 @@ class CheckClientDownloadRequest : public download::DownloadItem::Observer {
+                                   DownloadCheckResultReason* reason,
+                                   ClientDownloadRequest::DownloadType* type);
+ 
++ protected:
```

Comment:
I don't think these patches are necessary, but if we did need them they could be greatly reduced by just patching in `protected:` https://github.com/brave/brave-core/pull/2710/files#diff-050260a0b3f8a1a73c754196dd43744fR21 and then back to  `private:` it. There is no reason to move the methods themselves.

Reply:
actually for this particular usage the preferred method is to add a friend class for the subclass

Reply:
didn't notice that SendRequest was also made virtual, but we should add `virtual` on the line above the method to make it an addition instead of a modification

Reply:
actually I think a variation on @simonhong's suggestion would be the smallest patch here if we can't block in the network delegate (might not be able to if it uses the system request context) - but use a macro to make it one line - see https://github.com/brave/brave-core/pull/2010/commits/a80dd726a2052a591875d2e2d5b7260b82a157e6#diff-042dfd2d2e8c42e0acfe0b4028b2e897R60

Reply:
I rewrote the patch as a macro+override as suggested.